save-core/src/commonTest/kotlin/org/cqfn/save/core/PluginDiscoveryTest.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqfn.save.core
+
+import org.cqfn.save.core.config.SaveProperties
+import kotlin.test.Test
+
+class PluginDiscoveryTest {
+    @Test
+    fun `detect plugins`() {
+        val saveProperties = SaveProperties(
+            testRootPath = "../examples",
+        )
+
+        Save(saveProperties).performAnalysis()
```

Comment:
Since you already have this test, maybe you could add a couple of assertions on discovered plugins?

Reply:
plugin discovery - is private logic, we will test it later, after the logic wil appear